client/packages/system/src/Patient/PatientView/AddButton.tsx

```diff
@@ -34,13 +37,26 @@ export const AddButton = () => {
   >(options[1] as SplitButtonOption<PatientModal>);
 
   return (
-    <SplitButton
-      color="primary"
-      options={options}
-      selectedOption={selectedOption}
-      onSelectOption={setSelectedOption}
-      Icon={<PlusCircleIcon />}
-      onClick={() => setCurrent(selectedOption?.value)}
-    />
+    <>
+      <SplitButton
+        color="primary"
+        options={options}
+        selectedOption={selectedOption}
+        onSelectOption={setSelectedOption}
+        Icon={<PlusCircleIcon />}
+        onClick={() => setCurrent(selectedOption?.value)}
+      />
+      <ProgramSearchModal
```

Comment:
this might have to come out of this file when we add encounters etc. so far the file size is ok though, so I left it in.